Include consistent installation info (including Helm) across Access Request plugin docs

ptgott opened the pull request:
The Access Request plugin installation instructions are _almost_ standard. Make the instructions more consistent and easier to edit by replacing them with a partial. Use our partial parameter substitution syntax to supply the name of each plugin.

ptgott committed:
Use partial for Access Request plugin install docs

The Access Request plugin installation instructions are _almost_
standard. Make the instructions more consistent and easier to edit by
replacing them with a partial. Use our partial parameter substitution
syntax to supply the name of each plugin.

alexfornuto commented on docs/pages/includes/plugins/install-access-request.mdx:
````diff
@@ -0,0 +1,63 @@
+  $ make
+  ```
+
+  Run `./install` from `teleport-{{ name }}` or place the executable in the
````
I opened an issue in the plugins repo when testing this and found that the `install` script assumes the binary is in the CWD. I was [told](https://github.com/gravitational/teleport-plugins/pull/840#issuecomment-1570436183) that this script is to be used with the packaged tarballs, so this needs to be adjusted.

alexfornuto reviewed (changes requested):
One small issue.

ptgott commented on docs/pages/includes/plugins/install-access-request.mdx:
````diff
@@ -0,0 +1,63 @@
+development machine.
+
+  ```code
+  $ curl -L -O https://get.gravitational.com/teleport-access-{{ name }}-v(=teleport.plugin.version=)-linux-amd64-bin.tar.gz
````
I think it would make sense to decouple that work from this PR, since the aim of this PR is to refactor existing information.

alexfornuto replied:
Agreed, just pointing it out.

ptgott committed:
Respond to alexfornuto feedback

ptgott commented on docs/pages/includes/plugins/install-access-request.mdx:
````diff
@@ -0,0 +1,63 @@
+```code
+$ helm repo update
+```
+
````
The `helm install` command comes later in the guide, when we run the plugin

stevenGravy commented on docs/pages/includes/plugins/install-access-request.mdx:
```diff
@@ -0,0 +1,62 @@
+$ curl -L -O https://get.gravitational.com/teleport-access-{{ name }}-v(=teleport.plugin.version=)-linux-amd64-bin.tar.gz
+$ tar -xzf teleport-access-{{ name }}-v(=teleport.plugin.version=)-linux-amd64-bin.tar.gz
+$ cd teleport-access-{{ name }}
+$ ./install
```
This needs to be a `sud
```suggestion
$ sudo ./install
```

stevenGravy commented on docs/pages/access-controls/access-request-plugins/ssh-approval-msteams.mdx:
```diff
@@ -33,51 +33,7 @@ in your Teleport cluster.
 
 ## Step 2/9. Install the Teleport Microsoft Teams plugin
 
```
ms teams doesn't have the helm install.  We should either add the helm install or retain the current version.

ptgott replied:
Would you mind clarifying what you mean by "the helm install"? I see that there's a Helm chart available for the Microsoft Teams plugin, if that's what you're referring to: https://github.com/gravitational/teleport-plugins/tree/master/charts/access/msteams

stevenGravy commented on docs/pages/access-controls/access-request-plugins/ssh-approval-email.mdx:
```diff
@@ -36,21 +36,6 @@ in your Teleport cluster.
 
 ## Step 2/7. Install the Teleport email plugin
 
```
This doesn't have the helm install step like the msft one

stevenGravy commented:
Mentioned that msft and email doesn't have the helm example completed.  We can either complete them or leave for later. Should confirm all of them have the helm install step as part of testing.

ptgott commented:
TODO: 

It looks like the email and MS Teams guides are the only ones without a Helm installation step:

```
$ grep -RLE '\$ (helm upgrade|helm install)' docs/pages/access-controls/access-request-plugins/ssh-*
docs/pages/access-controls/access-request-plugins/ssh-approval-email.mdx
docs/pages/access-controls/access-request-plugins/ssh-approval-msteams.mdx
```

Since both of these have Helm charts, we should align them fully to the rest of the docs. I'll do that in this PR and use the partial for installation instructions.

- [ ] Email
- [ ] MS Teams

ptgott committed:
Add a missing `sudo`

Partially responds to stevenGravy feedback

ptgott committed:
Add consistent Helm details

Ensure all Access Request plugin guides include the same Helm-related
steps. Excluding the Jira plugin because that guide is out of date and
will be addressed in a separate change.

Also removes the "Feedback" section from each Access Request plugin
guide.  These are out of step with the rest of the docs, which don't
include a feedback section, and do not reflect the current location of
the Access Request plugin source.

ptgott commented:
@stevenGravy These should have a consistent level of detail in the Helm instructions now!

ptgott committed:
Spelling and linter fixes

ptgott commented:
@lsgunn-teleport @zmb3 Checking if one of you has time to review this one. Thanks!
